Dockerfile

```diff
@@ -30,10 +30,6 @@ RUN go mod download
 # Copy the sources
 COPY ./ ./
 
```

Comment:
Ups deleted my own comment :/

We need this for development: https://cluster-api.sigs.k8s.io/developer/tilt.html
Tilt uses it to live-reload the binary when rebuilding.

Reply:
Ah maybe it's not necessary anymore because it's added by tilt. I have to test it:
https://github.com/kubernetes-sigs/cluster-api/blob/master/Tiltfile#L127-L142

Reply:
Okay tested it. Seems like with v1alpha4 they solved this by using a separate dockerfile generated in the Tiltfile.

So while this is a breaking change for development with Tilt on master. It's fine for me as it won't be necessary after we hopefully merged the v1alpha4 PR soon. I also synced the Dockerfile with the upstream cluster-api Dockerfile on the v1alpha4 PR. There seem to be a lot more performance improvements there.: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/748/commits/2fc696d2f610dc920a08f5844b5aad8bbae38a20

Reply:
ok, I don't know it's from Tilt :)

thanks and it brings trouble to me (can't download those files and have to manually update dockerfile when build image)